Automattic/jetpack
# AI Assistant: Add chat history to inline extension

Fixes #37410
Fixes #37428
Depends on D149047-code

## Proposed changes:

* Adds a chat history, limited to the last 20 consecutive requests

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






To be tested with D149047-code

* Apply the diff above and sandbox the public API
* Enable debug logging
* On a Heading, send some consecutive requests
* Check the debug messages, see that each request adds the previous response and a new message on the array
* Check that the responses make sense
* Check that the content can be edited between requests and the current content is sent.

An example of history that was not possible before:
`Translate to: Portuguese.`
`Scratch that, go back to how it was before, but in all caps`
